pages/api-reference/connect/webhook-subscriptions/index.mdx

```diff
@@ -43,6 +43,35 @@ Webhooks are delivered in near real-time as events tied to your application are
+| Event Topic Name   | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| ------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| transfer:created   | **Description:** An account-to-account transfer was created between a main Dwolla Account and an External Party. <br /> **Timing:** Occurs upon a POST request to the [Initiate a transfer](https://developers.dwolla.com/api-reference/connect/transfers/initiate) endpoint when sending funds between a main Dwolla Account and an External Party.                                                                                                                                      |
+| transfer:pending   | **Description:** An account-to-account transfer was updated from created to pending. A pending status represents submission to a Financial Institution for processing. <br /> **Timing:** Occurs immediately following the transfer.created event when a transaction is submitted for processing.                                                                                                                                                                                         |
+| transfer:processed | **Description:** **Description:** An account-to-account transfer was updated from pending to processed. A processed status represents completed processing with the Financial Institution. <br /> **Timing:** Occurs when the transfer is expected to be successful based on the designated processing time.                                                                                                                                                                              |
+| transfer:failed    | **Description:** An account-to-account transfer was updated from pending to failed. <br /> **Timing:** Occurs after a transfer.pending event when a transaction fails to be submitted for processing. Additional details are available in an embedded `failureReason` object when retrieving a transfer.                                                                                                                                                                                  |
```

Comment:
Modified the description a bit, since the transfer is `submitted` but it failed processing.

```suggestion
| transfer:failed    | **Description:** An account-to-account transfer was updated from pending to failed. <br /> **Timing:** Occurs after a transfer:pending event when a transaction fails processing after being submitted to the Financial Institution. Additional details are available in an embedded `failureReason` object when retrieving a transfer.                                                                                                                                                                                  |
```